bundle/src/main/java/com/adobe/acs/commons/errorpagehandler/cache/impl/CacheEntry.java

```diff
@@ -39,6 +41,7 @@ public CacheEntry() {
         this.misses = new AtomicInteger();
         this.data = "";
         this.expiresAt = new Date(0);
+        this.contentType = "UTF-8";
```

Comment:
is the intention to hold the content type (mime type)? or the character encoding? Seems like maybe both, in which case they should be stored separately so that the encoding can be used in the `getBytes()` method.